tests/openwisp2/settings.py

```diff
@@ -5,7 +5,6 @@
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
 DEBUG = True
-TESTING = sys.argv[1] == 'test'
```

Comment:
why are you removing this, this is useful during development

Reply:
Unfortunately, I mistakenly duplicated this line in the `swapper` commit, `TESTING = sys.argv[1:2] == ['test']` exists 2 lines above this one. :smile:

Reply:
ah ok